en/appendices/4-0-upgrade-guide.rst

```diff
@@ -88,3 +88,10 @@ composer commands:
 
     php composer.phar require --dev --update-with-dependencies "phpunit/phpunit:^8.0"
     php composer.phar require --update-with-dependencies "cakephp/cakephp:4.0.*"
+    
+    
+    
+Update CakePHP Skeleton
+=======================
+
+It might also be helpful to upgrade the app skeleton of your cakePHP app. See https://packagist.org/packages/dereuromark/upgrade-cakephp for more information.
```

Comment:
Thats the wrong repo, and I dont think this helps without having a more clear doc on the actual commands needed probably.

What I was referring to was the missing info about the BodyParser middleware and anything concrete related that was missing on this page here.

Reply:
Ah ok. So it would be better to have a section on what should be looked at closer, when upgrading to cakephp4 in general? 
How can I update the documentary here the best way? Would I need to create a new pull request? Sorry, haven't had done something like this before -.-

Reply:
You can also change this PR

Reply:
Edit [/en/appendices/4-0-upgrade-guide.rst](https://github.com/gringlas/docs/blob/patch-4/en/appendices/4-0-upgrade-guide.rst) at your fork and commit to same `patch-4` branch, and changes will appear here in this PR.